src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -106,62 +113,67 @@ const handleClick = (): void => {
       cursor: not-allowed;
 
       .k-multiselect-item-label {
+        margin-right: var(--kui-space-40, $kui-space-40) !important;
         opacity: 0.6;
       }
     }
 
     .k-multiselect-item-label {
-      color: var(--grey-600);
-      font-size: 14px;
-      font-weight: 500;
-      line-height: 20px;
-      padding: 8px;
+      color: var(--grey-600, var(--kui-color-text-neutral-stronger, $kui-color-text-neutral-stronger));
+      font-size: var(--kui-font-size-30, $kui-font-size-30);
+      font-weight: var(--kui-font-weight-medium, $kui-font-weight-medium);
+      line-height: var(--kui-line-height-30, $kui-line-height-30);
+      padding: var(--kui-space-40, $kui-space-40);
       width: auto;
       word-break: break-word;
 
       :deep(.select-item-label) {
-        color: var(--grey-600);
-        font-size: 14px;
-        font-weight: 600;
-        margin-bottom: var(--spacing-xxs);
+        color: var(--grey-600, var(--kui-color-text-neutral-stronger, $kui-color-text-neutral-stronger));
+        font-size: var(--kui-font-size-30, $kui-font-size-30);
+        font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
+        margin-bottom: var(--spacing-xxs, var(--kui-space-20, $kui-space-20));
       }
 
       :deep(.select-item-desc) {
-        color: var(--grey-500);
-        font-size: var(--spacing-sm);
-        font-weight: 400;
+        color: var(--grey-500, var(--kui-color-text-neutral, $kui-color-text-neutral));
+        font-size: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
+        font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
       }
     }
     .selected-item-icon {
-      height: var(--spacing-lg);
+      height: var(--spacing-lg, var(--kui-space-80, $kui-space-80));
 
       &.kong-icon.kong-icon-lock {
         height: 14px;
-        padding-left: var(--spacing-xxs);
+        padding-left: var(--spacing-xxs, var(--kui-space-20, $kui-space-20));
+      }
+
+      &.is-item-selected {
+        color: $tmp-color-blue-200;
       }
     }
 
     .kong-icon:not(.selected-item-icon) {
-      margin-right: var(--spacing-sm);
+      margin-right: var(--spacing-sm, var(--kui-space-50, $kui-space-50));
     }
 
     .k-multiselect-selected-icon-container {
       height: var(--spacing-lg);
```

Comment:
What about this guy and the other references to `var(--spacing-lg)` for `width`?

Reply:
But won't the old spacing vars eventually go away? I'd think then, we want to hardcode the value to `24px` (or whatever it is) instead? Or at least provide a fallback?

Reply:
Just hard-code the value here as the fallback.

Reply:
Done